examples/VREP/dynamics_adaptation_osc.py

```diff
@@ -68,10 +79,6 @@
 
         # add an additional force for the controller to adapt to
         extra_gravity = robot_config.g(feedback['q']) * 2
```

Comment:
I don't think you need the `*2` since just adding gravity again is already doing double gravity, however it doesn't really matter, just takes longer to adapt

Reply:
yeah it's more of a visible effect so i added it in